.github/workflows/codeql-analysis.yml

```diff
@@ -100,6 +100,9 @@ jobs:
         sed -i -e "s%^# setup ssh service$%${command//$'\n'/\\n}\n\n# setup ssh service%" Dockerfile.test.?pu
 
         command=$(cat <<EOF
+        # Install recent CMake
```

Comment:
Why is installing `cmake` through `apt-get install`in the `Dockerfile`s not sufficient here?

Reply:
We need CMake 3.13 to compile the C++ parts of Horovod. On Ubuntu 18.04 and without extra package sources like a PPA from Kitware `apt-get install cmake` installs CMake 3.10 IIRC.

The mechanism of this PR does not work with the build for CodeQL because we don't use setup.py there.

Reply:
This can probably be removed again as we have moved to Ubuntu 20.04 for our test images in the meantime.

Reply:
I removed this and also went back to installing CMake via apt in the Dockerfiles based on Ubuntu 20.04 (`docker/*/Dockerfile`).